rttest/test/test_api.cpp

```diff
@@ -58,6 +59,28 @@ TEST(TestApi, read_args_get_params) {
   EXPECT_EQ(0, rttest_finish());
 }
 
+TEST(TestApi, read_args_update_period_over_32bit) {
+  // 4294967305 equals "static_cast<uint64_t>(UINT32_MAX) + 10"
+  // but avoid calculation to keep test simple
+  uint64_t update_period = 4294967305;
+  std::string update_period_str("4294967305ns");
```

Comment:
Small thing; if we ever change the number above, then this would get out of sync.  You can do this instead:

```
std::string update_period_str(std::to_string(update_period) + "ns");
```

Reply:
Exactly, thank you.